src/hooks/__tests__/useDebounceEvent.jest.ts

```diff
@@ -73,8 +73,11 @@ describe("useDebounceEvent", () => {
     it("should clear the value", () => {
       const { clearValue } = hookResult.result.current;
 
+      const syntheticChangeEvent = {
+        type: "change"
+      } as ChangeEvent<Partial<HTMLInputElement> | Partial<HTMLTextAreaElement>>;
       act(() => {
-        clearValue();
+        clearValue(syntheticChangeEvent);
       });
```

Comment:
probably can be reverted? or am I missing something?

Reply:
Yeah, just haven't pushed the changes after our talk - it's done now